DuckDuckGo/FindInPageView.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  FindInPageView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2019 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+class FindInPageView: UIView {
```

Comment:
There are some scenarios in which keyboard stays on the screen and whatever I type, goes nowhere. 

1. Open find in page. Search for something.
2. Open menu. Select 'Refresh page'.
3. Search bar is hidden, but keyboard is not.

or:
1. Open find in page. Search for something.
2. Long press on a link -> Open in new tab.
3. Search bar is hidden, but keyboard is not.

or:
1. Enable auto clear data with 5 min timer.
2. Open find in page. Search for something.
3. Leave the app for 5 min.
4. When back, tabs are cleared but keyboard is still up (and focus is not in omni bar).

Reply:
Fixed the first 2.  Still looking at the 3rd

Reply:
Ok, the 3rd one should be fixed now as well.